src/globals.ts

```diff
@@ -43,6 +43,35 @@ export const options: GlobalOptions = {
     timestamp: 0,
 };
 
+export const eventEmitter = new EventEmitter();
+export const minimapEvents = (() => {
```

Comment:
I would rather have minimap-specific features live in the minimap-folder instead of globaly